javaScriptLoops-forLoop/practiceProblems/consoleLogInt.js

```diff
@@ -0,0 +1,3 @@
+for (let i = 5; i <=100; i++) {
```

Comment:
Nonstandard spacing. Please add a space after comparison operators:
`for (let i = 5; i <= 100;  i++;) {`